web/source/kmwutils.ts

```diff
@@ -304,17 +304,12 @@ namespace com.keyman {
 
     // Found a bit of magic formatting that allows dynamic return typing for a specified element tag!
     _CreateElement<E extends "p"|"style"|"script"|"div"|"canvas"|"span">(nodeName:E) {
-      var e = document.createElement<E>(nodeName);
-
-      // Make element unselectable (Internet Explorer)
-      if (typeof e.onselectstart != 'undefined') { //IE route
-        e.onselectstart=this.selectStartHandler; // Build 360
-      } else { // And for well-behaved browsers (may also work for IE9+, but not necessary)
-        e.style.MozUserSelect="none";
-        e.style.KhtmlUserSelect="none";
-        e.style.UserSelect="none";
-        e.style.WebkitUserSelect="none";
-      }
+      const e = document.createElement<E>(nodeName);
+
+      e.style.MozUserSelect="none";
+      e.style.KhtmlUserSelect="none";
+      e.style.UserSelect="none";
+      e.style.WebkitUserSelect="none";
```

Comment:
Three of these are probably also unnecessary now but I haven't checked version support for *UserSelect?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

The fact that the article still references all of them and their differences is something I find concerning.